windows/sound.cpp

```diff
@@ -204,7 +196,7 @@ QString CSound::CheckDeviceCapabilities()
     }
 
     // special case with 4 input channels: support adding channels
-    if ( ( lNumInChan == 4 ) && bInputChMixingSupported )
+    if ( lNumInChan == 4 )

```

Comment:
Why did you remove the bInputChMixingSupported?

Reply:
Because all sample types now support the add channel.

Reply:
Great :-)